app/controllers/V1/submissions/searches_controller.rb

```diff
@@ -0,0 +1,137 @@
+module V1
+  module Submissions
+    class SearchesController < ApplicationController
+      def create
+        @data = search_query
```

Comment:
another option might be to extract theese private methods out to their own concern - maybe as a service class sending through `search_params` as an arg?

Reply:
I was thinking of doing this if it gets much more complex yes, but have left it as "shameless green" for now.